StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedFormViewController.swift

```diff
@@ -18,15 +18,13 @@ import UIKit
     /// This method is called when the user taps the primary button (e.g., "Buy") while `formSheetAction` is set to `.confirm`.
     /// - Parameters:
     ///   - embeddedFormViewController: The view controller requesting the confirmation.
-    ///   - paymentOption: The `PaymentOption` to be confirmed.
     ///   - completion: A completion handler to call with the `PaymentSheetResult` from the confirmation attempt.
     func embeddedFormViewControllerShouldConfirm(
         _ embeddedFormViewController: EmbeddedFormViewController,
-        with paymentOption: PaymentOption,
         completion: @escaping (PaymentSheetResult, STPAnalyticsClient.DeferredIntentConfirmationType?) -> Void
     )
 
-    /// This method is called when the user taps the primary button (e.g., "Buy") while `formSheetAction` is set to `.continue`.
+    /// This method is called when the user taps the primary button (e.g., "Buy") while `formSheetAction` is set to `.confirm` after confirmation.
```

Comment:
1. Maybe rewrite to:
When `formSheetAction` is set to `.confirm`, this method is called after a user taps the primary button (e.g, "Buy") followed by a successful confirmation.

2. The name of the function is awkward to me.  "xxxxShouldContinue" seems like the function would return a boolean to describe xxxx to continue or not.

3. The name "Continue" clashes with the formSheetAction value .continue -- maybe we can find a better name here?

Reply:
Renaming to `embeddedFormViewControllerDidCompleteConfirmation` and made a minor change to the doc comment.